scripts/install/configuration/apply_csm_configuration.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+usage()
+{
+   # Display Help
+   echo "Runs CFS to setup passwordless ssh"
+   echo "All parameters are optional and the values will be determined automatically if not set."
+   echo
+   echo "Usage: deploy_ssh_keys.sh [ --csm-release version ] [ --git-commit id ]"
+   echo "                            [ --git-clone-url url ] [ --ncn-config-file file ]"
+   echo "                            [ --xnames xname1,xname2... ] [ --clear-state ]"
+   echo
+   echo "Options:"
+   echo "csm-release          The version of the CSM release to use."
```

Comment:
Here is a case where you may want to throw in an e.g. in parenthesis, to give an example of what the CSM release argument looks like.

Reply:
Fixed